content/en/blog/2022/knative.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Distributed tracing in Knative
+date: 2022-04-12
+author: Pavol Loffay
+---
+
+In this article, you will learn how distributed tracing works in Knative and we will explore how the OpenTelemetry project can make tracing support in this environment easier. We will explore Knative under the hood to understand what distributed tracing capabilities it provides out-of-the-box and which parts of the system need additional instrumentation.
```

Comment:
would be good to link to the project when Knative is used for the first time.

Reply:
I am not sure I understand you. By project do you mean my "demo app" I referenced in this blog post?

Reply:
Sorry, my message was unclear, here's the markdown:
```markdown
In this article, you will learn how distributed tracing works in [Knative](https://knative.dev/) and we will 
```markdown